src/ds/ds-motion-common.cpp

```diff
@@ -514,11 +514,12 @@ namespace librealsense
             });
 
         bool high_sensitivity = _owner->is_gyro_high_sensitivity();
+        double gyro_scale_factor = high_sensitivity ? 0.003814697265625 : ( _fw_version >= firmware_version( 5, 15, 1, 224 ) ? 0.0001: 0.1 );    
```

Comment:
We should be able to get the factor from the feature, not check FW version here

Reply:
Also add a comment or a TODO about the HKR 0.003814697265625 value. We will need to support a unified scale factor between DS5 and HKR

Reply:
Added